process-dataset/steps/write-per-cell-jsons.js

```diff
@@ -24,15 +26,28 @@ const formatAndWritePerCellJsons = async (firebaseHandler, readFolder, outFolder
             const counts = {}
             for (let index = 0; index < json.length; index++) {
                 const cellData = json[index];
-                if (cellData.file_info.length !== FILE_INFO_KEYS.length) {
-                    console.error("file info in not in expected format")
+                let fileInfoKeys;
+                if (cellData.file_info.length === REQUIRED_FILE_INFO_KEYS.length) {
+                    fileInfoKeys = REQUIRED_FILE_INFO_KEYS
+                } else if (cellData.file_info.length === FILE_INFO_KEYS.length) {
+                    fileInfoKeys = FILE_INFO_KEYS
```

Comment:
What should we do if we add more optional items to fileInfo?

Reply:
we should include the array in the dataset then, instead of hardcoding it. Like we do with the features

Reply:
except that fileinfo is supposed to be standard info that every image has, as opposed to features which could be anything

Reply:
we could probably refactor to make this easier.  like not doing arrays for fileinfo (larger file size but easier to deal with?).  something to think about anyway.